tests/run_tests.sh

```diff
@@ -19,8 +19,8 @@ python_bin=$(command -v ${PYTHON_BIN})
 
 # check if python version meets our requirements
 IFS=" " read -r -a python_version <<< "$(${python_bin} -c 'import sys; print(sys.version_info[:])' | tr -d '(),')"
-if [ ${python_version[0]} -lt 3 ] || [[ "${python_version[0]}" -eq 3 && "${python_version[1]}" -lt 6 ]]; then
-    echo "Error: Unsupported python version \"${python_version[0]}.${python_version[1]}\". Requiring python 3.6 or higher."
+if [ ${python_version[0]} -lt 3 ] || [[ "${python_version[0]}" -eq 3 && "${python_version[1]}" -lt 7 ]]; then
+    echo "Error: Unsupported python version \"${python_version[0]}.${python_version[1]}\". Requiring python3."
```

Comment:
I think this line needs to be fixed.

```suggestion
    echo "Error: Unsupported python version \"${python_version[0]}.${python_version[1]}\". Requiring python 3.7 or higher."
```

Reply:
Sorry, missed that message. I'll fix it right now.

Reply:
@grische 

Is it already fixed?
It seems good for latest revision.

Reply:
Yes, it is fixed. From my point of view, I am ready to merge it.